Test/Exam1/src/exercice4/Gestion.java

```diff
@@ -0,0 +1,47 @@
+package exercice4;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class Gestion {
+	private final ArrayList<Reservation> reservation;
+	
+	public Gestion() {
+		reservation = new ArrayList<>();
+	}
+	
+	public void ajouterReservation(Reservation chamb) {
+		int idx = reservation.indexOf(chamb);
```

Comment:
Hmmm... This does not work unfortunately.
~~~java
Date d = new Date(/*some_date*/)
Reservation r1 = new Reservation("201",d);
Reservation r2 = new Reservation("201",d);
boolean b = r1 == r2 // b is false since objects' adresses are compared
~~~